packages/svelte-materialify/@types/Col.d.ts

```diff
@@ -1,6 +1,28 @@
 import { SvelteComponent } from 'svelte-materialify/@types/shared';
 
 interface ColProps {
+<<<<<<< HEAD
```

Comment:
This stuff should not be here

Reply:
I'm very sorry, it was late yesterday and I didn't notice that I didn't saved the resolved conflicts.

Reply:
no prob dude. Thanks 👍